sunpy/lightcurve/sources/noaa.py

```diff
@@ -162,25 +164,26 @@ class NOAAPredictIndicesLightCurve(LightCurve):
 
     """
 
-    def peek(self, **plot_args):
-        """Plots predicted NOAA Indices as a function of time. An example is shown below.
+    default_url_noaa_predict_indices = "http://services.swpc.noaa.gov/text/predicted-sunspot-radio-flux.txt"
 
-        .. plot::
+    def peek(self, **plot_args):
+        """
+        Plots predicted NOAA Indices as a function of time. An example is shown below.
 
-            from sunpy import lightcurve as lc
-            noaa = lc.NOAAPredictIndicesLightCurve.create()
-            noaa.peek()
+        >>> from sunpy import lightcurve as lc
+        >>> noaa = lc.NOAAPredictIndicesLightCurve.create()
+        >>> noaa.peek()
 
         Parameters
         ----------
         **plot_args : dict
-            Any additional plot arguments that should be used
-            when plotting.
+        Any additional plot arguments that should be used when plotting.
```

Comment:
yeah this indentation thing with travis in noaa file was bothering me a lot. let me fix the rest of the things you pointed out and commit so that it passes all test and then I'll fix these noaa file ones